packit/cloned_project.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import shutil
+
+import git
+
+from ogr.abstract import GitProject, GitService
+from packit.utils import is_git_repo, get_repo
+
+logger = logging.getLogger(__name__)
+
+
+class ClonedProject:
```

Comment:
I don't want to pick nits, but I really don't like the name :D I can live with it, but just saying

Reply:
I do not like it either. Do you have any idea?

We have GitProject/GitService from ogr and git.Repo from git...

Reply:
LocalProject, LocalGitRepo or some variant of it

Reply:
LocalProject :+1